### DISCUSSION TITLE
Is there some docs somewhere that shows how to get post/page Yoast SEO data?

### DISCUSSION THREAD

**Discussion by bmarshall511** [wpengine/faustjs#849] [Q&A] [answered]
Using Yoast for SEO (new to faustjs, so bear with me) & wondering how to get Yoast SEO data for a page/post.

**Comment by josephfusco** [comment, marked as answer]
Hey @bmarshall511, while we don't have 1:1 support with every feature in Yoast, we do support using sitemaps generated by Yoast which you can read about in our [Sitemap Guide](https://faustjs.org/docs/next/guides/sitemaps).

If you are looking for simple support for [Open Graph meta tags](https://developers.facebook.com/docs/sharing/webmasters/#markup), we have an [SEO component you can take from our basic Atlas Blueprint theme](https://github.com/wpengine/atlas-blueprint-basic/blob/main/src/components/SEO/SEO.js).
